src/libraries/Microsoft.PowerFx.Interpreter/Functions/LibraryOperators.cs

```diff
@@ -35,6 +35,14 @@ internal static partial class Library
             returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
             targetFunction: NumericDiv);
 
+        public static readonly AsyncFunctionPtr OperatorBinarySub = StandardErrorHandling<NumberValue>(
+            expandArguments: NoArgExpansion,
+            replaceBlankValues: ReplaceBlankWithZero,
+            checkRuntimeTypes: ExactValueType<NumberValue>,
+            checkRuntimeValues: FiniteChecker,
+            returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
+            targetFunction: NumericSub);
```

Comment:
to reduce copy & paste, I wonder if this could be wrapped in a helper like StandardMath, and then it's just

public static readonly AsyncFunctionPtr OperatorBinarySub = StandardMath(NumericSub);


And other operators here (like Add) could share that too.

Reply:
> to reduce copy & paste, I wonder if this could be wrapped in a helper like StandardMath, and then it's just
> 
> public static readonly AsyncFunctionPtr OperatorBinarySub = StandardMath(NumericSub);
> 
> And other operators here (like Add) could share that too.


Is this close of what you are looking for:

Microsoft.PowerFx.Interpreter\Functions\StandardErrorHandling.cs
```
private static AsyncFunctionPtr StandardMath<T>(
            Func<IRContext, T[], FormulaValue> targetFunction)
            where T : FormulaValue
        {
            return StandardErrorHandling<T>(NoArgExpansion, ReplaceBlankWithZero, ExactValueType<NumberValue>, FiniteChecker, ReturnBehavior.AlwaysEvaluateAndReturnResult, (runner, symbolContext, irContext, args) =>
            {
                var result = targetFunction(irContext, args);
                return new ValueTask<FormulaValue>(result);
            });
        }
```

Microsoft.PowerFx.Interpreter\Functions\LibraryOperators.cs

```
public static readonly AsyncFunctionPtr OperatorBinaryAdd = StandardMath<NumberValue>(
    targetFunction: NumericAdd);

public static readonly AsyncFunctionPtr OperatorBinaryMul = StandardMath<NumberValue>(
    targetFunction: NumericMul);

public static readonly AsyncFunctionPtr OperatorBinaryDiv = StandardMath<NumberValue>(
    targetFunction: NumericDiv);

public static readonly AsyncFunctionPtr OperatorBinarySub = StandardMath<NumberValue>(
    targetFunction: NumericSub);
```


All Sub functions now invert the right arg sign and call the respective Add function. 

```
private static FormulaValue SubDateAndTime(IRContext irContext, FormulaValue[] args)
{
    var args1 = (TimeValue)args[1];

    args[1] = new TimeValue(irContext, -args1.Value);

    return AddDateAndTime(irContext, args);
}
```